src/pages/exhibition/EnterScan.tsx

```diff
@@ -12,6 +12,25 @@ const EnterScan: React.VFC = () => {
   const aspida = useAspidaClient();
   const { currentUser, currentUserId } = useAuthState();
 
+  const [isFull, setIsFull] = useState<boolean>(false);
+
+  const checkIsFull = useCallback(async () => {
+    const status = await api(aspida)
+      .exhibitions._id(currentUser?.id || "")
+      .$get();
+    const sum = Object.entries(status.count)
+      .map(([, value]) => value)
+      .reduce((prev, curr) => prev + curr, 0);
+
+    setIsFull(sum >= status.capacity);
+  }, [aspida, currentUser?.id]);
+
+  useEffect(() => {
+    checkIsFull();
+    const intervalId = setInterval(checkIsFull, 15000);
+    return () => clearInterval(intervalId);
```

Comment:
↓じゃない？と思ったけどわたしもこう書いてたので文句は言えない…
挙動としてはたぶん変わらない……はず……！

```suggestion
    return () => {
      clearInterval(intervalId)
    };
```

Reply:
別に変えなくてもいい？いいよね